### PR TITLE
Concurrent refresh

### PR DESCRIPTION
**Pull request by protonate**

### CODE REVIEW

**Comment by ialex** [comment]
Im testing this branch using Partial Refresh with PulsePoint adapter causes and error on addBidResponse in the following line:

requestTimestamp: getBidSetForBidder(bid.bidderCode).start

This is the error: Uncaught TypeError: Cannot read property 'start' of undefined

This is the bid received by addBidResponse:

```
{
adId: "66517dac3823e259"
bidderCode: "pulsepoint"
getSize: ()
getStatusCode: ()
height: 0
statusMessage: "Bid returned empty or error response"
width: 0
}
```

File: src/prebid.js [hunk 1]
```diff
@@ -456,12 +496,12 @@ pbjs.registerBidAdapter = function (bidderAdaptor, bidderCode) {
   }
 };
 
-pbjs.bidsAvailableForAdapter = function (bidderCode) {
+pbjs.bidsAvailableForAdapter = function ({ auction, bidderCode }) {
```
Comment (protonate) [inline src/prebid.js]: don't use destructuring here it will break current implementation

**Comment by protonate** [comment]
Closing as work has moved to [Prebid Auctions](https://github.com/prebid/Prebid.js/tree/prebid-auctions) branch and a new PR will be opened from there.

**Comment by ialex** [comment]
@protonate do you have a TODO list for the auctions branch, i would like to help since we are using our custom solution now to run concurrent bids but we would rather to have official support on it, let me know where we would help you on that branch.

**Comment by protonate** [comment]
@ialex great, yes, thanks for the help. I've opened the Prebid Auctions PR #386 and working to complete dev on this today. Further updates are pending on this PR but please start taking a look at it to provide any feedback and confirm that these changes will accommodate your use case.

**Pull request closed without merging**